elements/pfe-navigation/src/pfe-navigation.js

```diff
@@ -178,6 +179,18 @@ class PfeNavigation extends PFElement {
     window.addEventListener("resize", this._debouncedPostResizeAdjustments, { passive: true });
     this._wasMobileMenuButtonVisible = this.isMobileMenuButtonVisible();
     this._wasSecondaryLinksSectionCollapsed = this.isSecondaryLinksSectionCollapsed();
+
+    // Initial position of this element from the top of the screen
+    this.top = this.getBoundingClientRect().top || 0;
+
+    // If the nav is set to sticky, run the sticky handler and attach scroll event to window
+    if (this.hasAttribute("pfe-sticky") && this.getAttribute("pfe-sticky") != "false") {
+      // Run the sticky check on first page load
+      this._stickyHandler();
+
+      // Attach the scroll event to the window
+      window.addEventListener("scroll", this._stickyHandler);
```

Comment:
Can you throttle this with requestAnimationFrame, e.g. https://wiki.developer.mozilla.org/en-US/docs/Web/API/Document/scroll_event#Scroll_event_throttling

Reply:
@wesruv added those changes.